PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -65,55 +64,9 @@ class PayPalClient @VisibleForTesting internal constructor(
         payPalRequest: PayPalRequest,
         callback: PayPalPaymentAuthCallback
     ) {
-        braintreeClient.sendAnalyticsEvent(PayPalAnalytics.TOKENIZATION_STARTED)
-        if (payPalRequest is PayPalCheckoutRequest) {
-            isVaultRequest = false
-            sendCheckoutRequest(context, payPalRequest, callback)
-        } else if (payPalRequest is PayPalVaultRequest) {
-            isVaultRequest = true
-            sendVaultRequest(context, payPalRequest, callback)
-        }
-    }
-
-    private fun sendCheckoutRequest(
-        context: Context,
-        payPalCheckoutRequest: PayPalCheckoutRequest,
-        callback: PayPalPaymentAuthCallback
-    ) {
-        braintreeClient.sendAnalyticsEvent("paypal.single-payment.selected", analyticsParams)
-        if (payPalCheckoutRequest.shouldOfferPayLater) {
-            braintreeClient.sendAnalyticsEvent(
-                "paypal.single-payment.paylater.offered",
-                analyticsParams
-            )
-        }
-
-        braintreeClient.getConfiguration { configuration: Configuration?, error: Exception? ->
-            if (error != null) {
-                callbackCreatePaymentAuthFailure(callback, PayPalPaymentAuthRequest.Failure(error))
-            } else if (payPalConfigInvalid(configuration)) {
-                callbackCreatePaymentAuthFailure(
-                    callback,
-                    PayPalPaymentAuthRequest.Failure(createPayPalError())
-                )
-            } else {
-                sendPayPalRequest(context, payPalCheckoutRequest, callback)
-            }
-        }
-    }
+        isVaultRequest = payPalRequest is PayPalVaultRequest
 
-    private fun sendVaultRequest(
-        context: Context,
-        payPalVaultRequest: PayPalVaultRequest,
-        callback: PayPalPaymentAuthCallback
-    ) {
-        braintreeClient.sendAnalyticsEvent("paypal.billing-agreement.selected", analyticsParams)
-        if (payPalVaultRequest.shouldOfferCredit) {
-            braintreeClient.sendAnalyticsEvent(
-                "paypal.billing-agreement.credit.offered",
-                analyticsParams
-            )
-        }
+        braintreeClient.sendAnalyticsEvent(PayPalAnalytics.TOKENIZATION_STARTED, analyticsParams)
```

Comment:
Calling out that I moved this analytics call to after `isVaultRequest` is set. I also added the passing of `analyticsParams`.

We were previously omitting if the analytics event is a vault request.